src/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Hospice Hope</title>
+    <meta
+      name="description"
+      content="Learn how Hospice Hope has helped people with life limiting conditions"
+    />
+    <meta property="og:title" content="Hospice Hope" />
+    <meta
+      name="og:description"
+      content="Learn how Hospice Hope has helped people with life limiting conditions"
+    />
+    <meta
+      property="og:image"
+      content="https://neontribe.github.io/contemplating-action/banner.png"
```

Comment:
If people share a link, this will give them the ca-core banner image. I think we can re-card for when the hosting for this instance is set

Reply:
Yeah, @katjam mentioned the same issue and said we would need to look at it later

Reply:
Carded here: https://github.com/neontribe/ca-hospice-hope/issues/42